Packs/CiscoStealthwatch/Integrations/CiscoStealthwatch/cisco_stealthwatch.yml

```
@@ -0,0 +1,410 @@
+      isArray: false
+      name: search_id
+      required: true
+      secret: false
+    deprecated: false
+    description: List security events status
+    execution: false
+    name: cisco-stealthwatch-list-security-events-status
+    outputs:
+    - contextPath: CiscoStealthwatch.SecurityEventStatus.id
+      description: The id of the SecurityEvent
+      type: str
+    - contextPath: CiscoStealthwatch.SecurityEventStatus.percentComplete
+      description: The percentCompleteof the SecurityEvent
+      type: str
+  - arguments:
+    - default: false
+      description: The tenant we want to retrieve its list security events results
+      isArray: false
+      name: tenant_id
+      required: true
+      secret: false
+    - default: false
+      description: The id of the search from the initialize command
+      isArray: false
+      name: search_id
+      required: true
+      secret: false
+    - default: true
+      defaultValue: '50'
+      description: security events limit
+      isArray: false
+      name: limit
+      required: true
+      secret: false
+    deprecated: false
+    description: List security events results, use this command after the search job
+      is over.
+    execution: false
+    name: cisco-stealthwatch-list-security-events-results
+    outputs:
+    - contextPath: CiscoStealthwatch.SecurityEventResults.id
+      description: The id of the SecurityEvent
+      type: str
+    - contextPath: CiscoStealthwatch.SecurityEventResults.domainId
+      description: The domainId of the SecurityEvent
+      type: str
+    - contextPath: CiscoStealthwatch.SecurityEventResults.deviceId
+      description: The deviceId of the SecurityEvent
+      type: str
+    - contextPath: CiscoStealthwatch.SecurityEventResults.securityEventType
+      description: The securityEventType of the SecurityEvent
+      type: str
+    - contextPath: CiscoStealthwatch.SecurityEventResults.firstActiveTime
+      description: The firstActiveTime of the SecurityEvent
+      type: str
+    - contextPath: CiscoStealthwatch.SecurityEventResults.lastActiveTime
+      description: The lastActiveTime of the SecurityEvent
+      type: str
+    - contextPath: CiscoStealthwatch.SecurityEventResults.source
+      description: The source of the SecurityEvent
+      type: str
+    - contextPath: CiscoStealthwatch.SecurityEventResults.target
+      description: The target of the SecurityEvent
+      type: str
+    - contextPath: CiscoStealthwatch.SecurityEventResults.details
+      description: The details of the SecurityEvent
+      type: str
+    - contextPath: CiscoStealthwatch.SecurityEventResults.hitCount
+      description: The hitCount of the SecurityEvent
+      type: str
+  dockerimage: demisto/python3:3.9.1.15759
+  feed: false
+  isfetch: false
+  longRunning: false
+  longRunningPort: false
+  runonce: false
+  script: '-'
+  subtype: python3
+  type: python
+fromversion: 5.5.0
```

Please add the test playbook here like that: 

```suggestion
tests:
  - Yout test playbook ID
fromversion: 5.5.0
```